### ISSUE TITLE
manifest/variablesCustom - broken CSS variables

### ISSUE THREAD

**Issue by aheissenberger** [infinum/eightshift-frontend-libs#701]
### Describe your bug

**multiple** CSS variables defined in the `variablesCustom` section of a `manifest.json` will be inlined in the backend block editor without a delimiter which breaks their definition:
`--button-scoped-border-radius: 0.25rem--button-scoped-padding-desktop: .31640625rem .84375rem`

### Steps to Reproduce

custom/button/manifest.json

when I add a this section:
```json
"variablesCustom": [
		"--button-scoped-border-radius: 0.25rem",
		"--button-scoped-padding-desktop: .31640625rem .84375rem"
	]
```

it will work on frontend, but in the editor, it will inline without a delimiter which breaks the variables:
`--button-scoped-border-radius: 0.25rem--button-scoped-padding-desktop: .31640625rem .84375rem`

**my temporal fix was to add a semicolon:**
```json
"variablesCustom": [
		"--button-scoped-border-radius: 0.25rem;",
		"--button-scoped-padding-desktop: .31640625rem .84375rem;"
	]
```

### Expected behavior

inlined with a delimiter which is allowed in CSS to separate statements e.g. `;`

### Environment info

- @eightshift/frontend-libs: 8.1.0
- infinum/eightshift-libs: 6.4.0
- node: v18.14.2

### Please confirm that you have searched existing issues in this repo.

Yes

### Please confirm that you have searched in our documentation and couldn't find the answer.

Yes

### Please confirm that your bug occurs with all plugins uninstalled and with the default WordPress theme active.

Yes

**Comment by iruzevic** [comment]
@goranalkovic-infinum can you check this. It should be an easy fix

**Comment by goranalkovic-infinum** [comment]
Fixed in Frontend libs 8.2!
